.github/workflows/registry.yml

```diff
@@ -87,23 +91,30 @@ jobs:
       - name: Tag helm chart image
         run:  |
           version=${{ steps.get_version.outputs.VERSION }}
-          sed -i "s/latest/$version/g" charts/vela-core/values.yaml
+          sed -i "s/latest/$version/g" $HELM_CHART/values.yaml
```

Comment:
do we also need to sed the legacy chart for this version?

Reply:
You are right. As changed `values.yaml` will be synchronized to the legacy chart, so I didn't change it directly.
```
      - name: Prepare legacy chart
        run: |
          rsync -r $LEGACY_HELM_CHART $HELM_CHARTS_DIR
          rsync -r $HELM_CHART/* $LEGACY_HELM_CHART --exclude=Chart.yaml --exclude=crds
```